packages/core/src/rum/instrumentation/DdRumResourceTracking.tsx

```diff
@@ -176,32 +176,65 @@ const generateTracingAttributes = (
     };
 };
 
+const getFirstPartyHostsRegex = (firstPartyHosts: string[]): RegExp => {
+    if (firstPartyHosts.length === 0) {
+        return new RegExp('a^');
```

Comment:
That's a good question.
In terms of architecture I think I prefer to handle the case where the hosts are not specified only once in my code. With this solution I don't have to check whether the regexp is `getTracingAttributes` in my next function.

I think this is a trade-off between readability (it is a bit less readable) and ease of use.
So I am ok to extract it into a named constant as it will improve readability.